tests/draw/test_table.py

```diff
@@ -1517,3 +1517,427 @@ def test_running_elements_table_border_collapse_margin(assert_pixels):
+      _fffffqqKKqqqqKqqqqKKqqqqKqqllllllyyKKyyyyKyyyyKKyyyyKyyyyyyyyyyKKyyyyKyyyyKKyyyyKyyyyyyyyyyKKyyyyKyyyyKKyyyyKyyoooooo__
+      _fffffqqqKKqqKqqqqqqKKqqKqqqllllllyyyKKyyKyyyyyyKKyyKyyyyyyyyyyyyKKyyKyyyyyyKKyyKyyyyyyyyyyyyKKyyKyyyyyyKKyyKyyyoooooo__
+      _fffffqqqqKKKqqqqqqqqKKKqqqqllllllyyyyKKKyyyyyyyyKKKyyyyyyyyyyyyyyKKKyyyyyyyyKKKyyyyyyyyyyyyyyKKKyyyyyyyyKKKyyyyoooooo__
+      _fffffqqqqKKKqqqqqqqqKKKqqqqllllllyyyyKKKyyyyyyyyKKKyyyyyyyyyyyyyyKKKyyyyyyyyKKKyyyyyyyyyyyyyyKKKyyyyyyyyKKKyyyyoooooo__
+      _fffffqqqqqKKqqqqqqqqqKKqqqqllllllyyyyyKKyyyyyyyyyKKyyyyyyyyyyyyyyyKKyyyyyyyyyKKyyyyyyyyyyyyyyyKKyyyyyyyyyKKyyyyoooooo__
+      _fffffqqqqKqKKqqqqqqqKqKKqqqllllllyyyyKyKKyyyyyyyKyKKyyyyyyyyyyyyyKyKKyyyyyyyKyKKyyyyyyyyyyyyyKyKKyyyyyyyKyKKyyyoooooo__
+      _fffffqqqKqqKKqqqqqqKqqKKqqqllllllyyyKyyKKyyyyyyKyyKKyyyyyyyyyyyyKyyKKyyyyyyKyyKKyyyyyyyyyyyyKyyKKyyyyyyKyyKKyyyoooooo__
+      _fffffqqqKqqqKKqqqqqKqqqKKqqllllllyyyKyyyKKyyyyyKyyyKKyyyyyyyyyyyKyyyKKyyyyyKyyyKKyyyyyyyyyyyKyyyKKyyyyyKyyyKKyyoooooo__
+      _fffffqqKqqqqqKKqqqKqqqqqKKqllllllyyKyyyyyKKyyyKyyyyyKKyyyyyyyyyKyyyyyKKyyyKyyyyyKKyyyyyyyyyKyyyyyKKyyyKyyyyyKKyoooooo__
+      _fffffKKKKqqqKKKKKKKKqqqKKKKllllllKKKKyyyKKKKKKKKyyyKKKKyyyyyyKKKKyyyKKKKKKKKyyyKKKKyyyyyyKKKKyyyKKKKKKKKyyyKKKKoooooo__
+      _fffffqqqqqqqqqqqqqqqqqqqqqqllllllyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyoooooo__
+      _fffffqqqqqqqqqqqqqqqqqqqqqqllllllyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyoooooo__
+      _fffffqqqqqqqqqqqqqqqqqqqqqqllllllyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyoooooo__
+      _fffffqqqqqqqqqqqqqqqqqqqqqqllllllyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyoooooo__
+      _fffffqqqqqqqqqqqqqqqqqqqqqqllllllyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyoooooo__
+      ________________________________________________________________________________________________________________________
+    ''', '''
+      <style>
+        @font-face { src: url(weasyprint.otf); font-family: weasyprint }
+        @page {
+          margin: 1px;
+          size: 120px 56px;
+          @bottom-center { content: element(table); width: 100% }
+        }
+        table {
+         border-collapse:collapse;
+         margin-bottom: 1em;
+         }
+         tr {
+         border-left: 5px solid fuchsia;
+         border-right: 5px solid purple;
+         }
+         td.right {
+         background-color: yellow;
+         border-right: 5px solid orange;
+         }
+         td.left {
+         background-color: aqua;
+         border-right: 5px solid teal;
+         }
+      </style>
+      <table dir="ltr">
+        <tr>
+        <td class="left">XX</td>
+        <td class="right">XX XX XX</td>
+        </tr>
+      </table>
+
+
+      <table style="direction: rtl;">
```

Comment:
Use `direction` CSS property to set the value since `dir` attribute is not supported